sweep/sweeper.go

```diff
@@ -296,6 +296,12 @@ type UtxoSweeperConfig struct {
 	//   #1: min = 1 sat/vbyte, max (exclusive) = 11 sat/vbyte
 	//   #2: min = 11 sat/vbyte, max (exclusive) = 21 sat/vbyte...
 	FeeRateBucketSize int
+
+	// CancelRebroadcast is used to inform the rebroadcaster sub-system
+	// that it no longer needs to try to rebroadcast a transaction. This is
+	// used to ensure that invalid transactions (inputs spent) aren't
+	// retried in the background.
+	CancelRebroadcast func(tx chainhash.Hash)
```

Comment:
Should this be in the `Wallet` interface under `sweep/interface.go`?

Reply:
Sure that makes sense, I can move it there.